src/RemoteApi/Commands/GetBalances/Request.php

```diff
@@ -72,7 +72,7 @@ public function setAddresses(array $addresses): self
      */
     public function addAddress(Address $address): self
     {
-        $this->addresses[] = $address->removeChecksum();
```

Comment:
`removeChecksum` returns a new instance of the address without a checksum.

Reply:
See other comment, I prefer to refactor it and put it at an other point of the code.